pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -337,8 +337,6 @@ func (c *controller) List(typ config.GroupVersionKind, namespace string) ([]conf
 		}
 
 		switch typ {
-		case gvk.VirtualService:
```

Comment:
This code is still used

Reply:
hmm, this is a little bit confuse to me, since it depends on the convert ingress to virtual service code, that's why i delete it. I might need to look into it more.